NewTutorials/Tutorial/SetupTutorials.swift

```diff
@@ -4,24 +4,95 @@ import UIKit
 import OneMobileSDK
 import PlayerControls
 
-func setupPlayingVideos(vc: TutorialCasesViewController) {
-    func process(player: Future<Result<Player>>) {
-        guard let wrapper = vc.storyboard?.instantiateViewController(withIdentifier: "PlayerViewControllerWrapper") as? PlayerViewControllerWrapper else {
-            fatalError("Wrapper not found")
-        }
-        
-        vc.show(viewController: wrapper)
-        wrapper.props.player = player
+
+func show(wrapper: PlayerViewControllerWrapper,
+          in tutorialCasesViewController: TutorialCasesViewController,
+          with player: Future<Result<Player>>) {
+    tutorialCasesViewController.show(viewController: wrapper)
```

Comment:
Can wrapper be created inside this function?

Reply:
No reason to pass it each time to this function.

Reply:
The same about `TutorialCasesViewContoller` - maybe function `show` should be maid inside this class?